service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -1439,6 +1579,7 @@ private String CreateImportJobPropertiesJson(String importBlobContainerUri, Stri
         return jobProperties.toJobPropertiesParser().toJson();
     }
 
+
```

Comment:
in c# we don't use multiple empty lines between class members. I don't now what Java's style guidelines are, but here's a note for you to check on that.